app/medInria/medFilteringWorkspace.cpp

```diff
@@ -44,6 +44,7 @@ medFilteringWorkspace::medFilteringWorkspace(QWidget *parent): medAbstractWorksp
 {
     d->filteringToolBox = new medFilteringSelectorToolBox(parent);
     connect(d->filteringToolBox,SIGNAL(processFinished()),this,SLOT(onProcessSuccess()));
+    d->filteringToolBox->setWorkspace(this);
```

Comment:
Yes, because it's a missing code :), we need to have access to workspace in toolboxes